src/app/pages/server-listing/server-listing.component.ts

```
@@ -510,4 +646,39 @@ export class ServerListingComponent implements OnInit, OnDestroy {
     return item.display_name !== item.api_reference && item.display_name !== item.api_reference.replace('Standard_', '');
   }
 
+  openModal() {
+    this.modalText = 'Select Benchmark Category';
+    this.modalBenchmarkSelect.show();
+  }
+
+  closeModal(confirm: boolean) {
```

ESLint

[error] 'confirm' is defined but never used.

(@typescript-eslint/no-unused-vars)